script/lib/utils.js

```diff
@@ -1,4 +1,4 @@
-const OUT_DIR = process.env.ELECTRON_OUT_DIR || 'Debug'
+const OUT_DIR = process.env.ELECTRON_OUT_DIR || 'Default'
```

Comment:
This is just a temporary commit, to run tests in this branch. Will revert it once ci is fixed in master.